docs/articles/nunit/running-tests/NUnitLite-Runner.md

```diff
@@ -16,22 +16,31 @@ With NUnit 3.0, the NUnitLite framework has been merged with the full NUnit fram
 
 To run tests under NUnitLite, proceed as follows:
 
-* Create a console application to use as your test assembly. If you have already placed tests in a dll, you can change
-  the project so that it produces a console application and reload it.
+* You can use your existing unit test project (created with the `dotnet new nunit` command or from Visual Studio) or
+ create a new console application.  If you use an existing test project you will get a warning about duplicate entry points. This can be ignored.
 
-* Make sure your test assembly references both **nunit.framework** and **nunitlite**.
+Using a standard test project has the benefit that you can use both the standard `dotnet test` command, the Visual
+ Studio Test Explorer and the NUnitLite runner to run your tests.
 
-* The Main() for your console application should look something like this...
+* Make sure your test assembly references both **nunit.framework** and **nunitlite**.
 
-```csharp
-public static int Main(string[] args)
-{
-  return new AutoRun().Execute(args);
-}
+```xml
```

Comment:
ℹ️ I ended up having to change this back because it looks like docfx has issues pulling in csproj files or xml files or something along those lines. Everything I tried came up blank.